### DISCUSSION TITLE
tranimeizle

### DISCUSSION THREAD

**Discussion by ghost** [KebabLord/turkanime-indirici#43] [Genel]
Site yapısı turkanime'ye benzediği için entegre etmesi çok zaman almaz gibi duruyor tabi bu bir varsayım sadece, türkanimede olmayan animelere erişmek için güzel olabilir, anizm'de olabilir ama onun bölüm yapısı falan farklıydı bir ara ondan bilmiyorum onu.

**Comment by KebabLord** [comment]
hakikaten şuana dek diğer alternatifler hakkında hiç düşünmedim, uygulamayı en başında türkanime üstüne inşa etmemin sebebi bilinen en eski ve en büyük türkçe anime streamer servisi oluşuydu. Ama bunca zamandır bunun cloudflare önlemini bypass edelim, değişikliklerine adapte edelim diye uğraşacağımıza anizm gibi diğer alternatifleri de değerlendirebilirmişiz. Çünkü şu su götürmez bir gerçek ki eğer ortada bypass edilmesi gereken bir cloudflare koruması yoksa bu scripti hem çok daha hızlı, hem çok daha taşınabilir hem de çok daha hafif kılar. Şuanlık anizm ilgimi çekti, ama cloudflare'i ara sıra açıyorlar mı, kullanıyorlar mı emin olmadan da atılmak büyük zaman israfına yol açabilir

tranime'nin captcha'sı uğraşılmaya değmez
![image](https://user-images.githubusercontent.com/24779257/214442077-848f9b40-4c47-46a8-a127-c9c4c9542c46.png)

bu arada türkanime'de olmayan ancak tranime'de olan seriler ne kadar büyük? İkisinin veritabanını kıyaslayacak olsan hangisi derdin?

**Reply by ghost** [reply]
Seri olarak aklıma şuan Denpa Kyoushi geldi bir tek o da öyle pek büyük bir anime değil, database olarak yüksek ihtimal türkanime daha büyüktür hepsinden ama anizm ile kafa kafaya idiler diye hatırlıyorum bir zamanlar, bir de yanlis hatirlamiyorsam güncel serilerde anizm turkanimeye göre daha yavaştı

**Comment by rifatozkancomtr** [comment]
turkanime çok geç guncelleniyor ama büyük ihtimalle en büyük havuz yine onlarda var.
tranimeci var fakat altyazılar yapay zeka ile  hazırlanmış gibi duruyor baya baya.

Kendimiz yaparsak
https://9anime.to/ en büyük anime sitelerinden biri.
https://github.com/jhj0517/Whisper-WebUI sanırım bu altyazı çeviri için

:) fikir sadece.

**Reply by ghost** [reply]
Whisprer cevriyi sadece baska dilden inge ceviriyor

**Reply by rifatozkancomtr** [reply]
Hımm vardır belki alternatif yapay zeka ile çevrilen altyazılar oldukça iyi oluyor çünkü. 

4354 items sadece anime dizi olarak olanların sayısı. 9animede :)

**Reply by ghost** [reply]
Selenium ile deepl nin sitesini scrapeleyebilirsek iyi cunku eng tr ceviride en iyisi onlar

**Reply by rifatozkancomtr** [reply]
Openai ile yapamaz mıyız? 
Daha fazla metni tek seferde daha sorunsuz çeviririz gibime geliyor.

**Reply by ghost** [reply]
Gpt cevirisi parali ve gpt 3.5 ceviri konusunda kotu, gpt 4 iyidir belki ama onun apisi daha da pahalidir

**Reply by rifatozkancomtr** [reply]
https://github.com/Twoure/9anime.bundle

Plex için eklentisi bile var aslında baya geliştirmişler.
Chatgpt için apili hesaplar satıyorlardı en son ggsell de yada r10 da uygun fiyatlı :)

**Reply by ghost** [reply]
9animedeki videolar hardsub illa kendimiz ceviri yapacaksak torrentten cekmeli

**Reply by rifatozkancomtr** [reply]
hardsub dediğin göümü altyazı dimi :) kusura bakma o kadar bilgim yok kendi anime arşivim var nas sunucuda tutuyorum ama yeterli gelmiyor biraz takıntılıyım ahaha video playerlerinde alt yazı açma kapatma olduğu için dışardan yüklendiği düşündüm.

altyazıalr gömülü değil büyük ihtimalle sonradan ekleniyor. 
yani teknik bilgim yok ama player den indirilebilir sanırım yada kazınabilir :)

https://github.com/Fantasy-Boss/9anime-Automated-Downloader
